### ISSUE TITLE
Don't know how to create ISeq from: clojure.lang.Symbol

### ISSUE THREAD

**Issue by residentsummer** [tengstrand/lein-polylith#9]
**Describe the bug**
While following the guide to migrate an existing project to polylith, I've copied all the source into an empty base, then run "sync" command. Here is the output:

```
clj$> sync
updated: environments/development/project.clj
updated: bases/site-api/project.clj
updated: systems/site-backend/project.clj
Don't know how to create ISeq from: clojure.lang.Symbol
clj$> sync
Don't know how to create ISeq from: clojure.lang.Symbol
```

After that, `sync` and most of other commands produced just the error message.

**To Reproduce**
1. Create a new workspace, system and a base.
2. In a base's `core.clj` add a `require` statement like that:
    ```
    (ns com.example.site-api.core
      (:require clojure.string)
      (:gen-class))
    ```
3. Save, `lein polylith sync` => error
4. Wrap `clojure.string` in a pair of square brackets:
    ```
    (ns com.example.site-api.core
      (:require [clojure.string])
      (:gen-class))
    ```
5. Save, `lein polylith sync` => no error

**Expected behavior**
`sync` should run without an error.

If it's not possible with current design, plugin should provide a clue of what's wrong and where (path to "broken" file). To debug this I've copied files one by one to find the culprit.

**Operating System (please complete the following information):**
 - OS: MacOS
 - Version 10.13.6

**Versions (please complete the following information):**
 - Leiningen: 2.8.1
 - Java: 1.8.0_112 Java HotSpot(TM) 64-Bit Server VM
 - Polylith: Polylith 0.1.1 (2018-10-02)

**Comment by residentsummer** [comment]
By the way, thanks for developing Polylith! Documentation is really great (and video is awesome as well) and I'm eager to test how it will help to streamline the development process.

**Comment by tengstrand** [comment]
Yes, this looks like a proper bug.
Good job that you managed to narrow it down.
I will fix this in the next release, 0.1.2.

I'm glad that you appreciate the documentation!

If you find any other problems, don't hesitate to contact us.
